pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -666,43 +627,37 @@ func newComputeSlopeV1Hint(point0, point1 hinter.ResOperander) hinter.Hinter {
 			if err != nil {
 				return err
 			}
-			point0MemoryValues, err := vm.Memory.GetConsecutiveMemoryValues(point0Addr, int16(6))
+
+			point1Addr, err := point1.GetAddress(vm)
 			if err != nil {
 				return err
 			}
 
-			point1Addr, err := point1.GetAddress(vm)
+			point0XValues, err := vm.Memory.ResolveAsBigInt3(point0Addr)
+			if err != nil {
+				return err
+			}
+
+			point1XValues, err := vm.Memory.ResolveAsBigInt3(point1Addr)
 			if err != nil {
 				return err
 			}
+
+			point0MemoryValues, err := vm.Memory.GetConsecutiveMemoryValues(point0Addr, int16(6))
+			if err != nil {
+				return err
+			}
+
 			point1MemoryValues, err := vm.Memory.GetConsecutiveMemoryValues(point1Addr, int16(6))
 			if err != nil {
 				return err
 			}
 
-			// [x.d0, x.d1, x.d2]
-			var point0XValues [3]*fp.Element
 			// [y.d0, y.d1, y.d2]
 			var point0YValues [3]*fp.Element
-			// [x.d0, x.d1, x.d2]
-			var point1XValues [3]*fp.Element
 			// [y.d0, y.d1, y.d2]
 			var point1YValues [3]*fp.Element
 
-			for i := 0; i < 3; i++ {
-				point0XValue, err := point0MemoryValues[i].FieldElement()
-				if err != nil {
-					return err
-				}
-				point0XValues[i] = point0XValue
-
-				point1XValue, err := point1MemoryValues[i].FieldElement()
-				if err != nil {
-					return err
-				}
-				point1XValues[i] = point1XValue
-			}
-
 			for i := 3; i < 6; i++ {
```

Comment:
Doesn't it make sense to replace the parsing of y values too?

`point0YAddr` would just be `point0Addr.AddOffset(3)`. The we can just use that address on `ResolveAsBigInt3` to parse the y values too?

Reply:
@har777  is right, you can also refactor here

Reply:
I thought so too, but doing that resulted in test fails.

Reply:
can you show what you tried?

Reply:
I just tested locally and it works well:

```
	point0AdjustedAddr, err := point0Addr.AddOffset(3)
			if err != nil {
				return err
			}

			point0YValues, err := vm.Memory.ResolveAsBigInt3(point0AdjustedAddr)
			if err != nil {
				return err
			}
```

Reply:
the name of variables is probably not ideal with 'adjusted'

Reply:
> Doesn't it make sense to replace the parsing of y values too?
> 
> `point0YAddr` would just be `point0Addr.AddOffset(3)`. The we can just use that address on `ResolveAsBigInt3` to parse the y values too?

@har777 @TAdev0 I misunderstood this comment, working on the changes now.